scripts/install

```diff
@@ -66,19 +66,40 @@ port=$(ynh_find_port 3000)	# Cherche un port libre.
 ynh_app_setting_set $app port $port
 
 #=================================================
-# INSTALL DEPENDENCIES
+# INSTALL NODEJS
 #=================================================
+ynh_install_nodejs 8.0.0
```

Comment:
I didn't find where the required version was specified in cryptpad doc; any URL?